estimation_tracker/tracker/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+# Create your models here.
+
+class Task(models.Model):
+    task_name = models.TextField(blank=False)
```

Comment:
Related question: do you know the difference between `blank=True` and `null=True`?